e2e-tests/production-runtime/package.json

```
@@ -30,7 +30,7 @@
   ],
   "license": "MIT",
   "scripts": {
-    "build": "cross-env CYPRESS_SUPPORT=y gatsby build",
+    "build": "cross-env GATSBY_PREFIXED_FROM_COMMAND_LINE=YES FROM_COMMAND_LINE=YES CYPRESS_SUPPORT=y gatsby build",
     "build:offline": "cross-env TEST_PLUGIN_OFFLINE=y CYPRESS_SUPPORT=y gatsby build",
```

I think we need to add env vars to `build:offline` script variant as well. We run couple different builds in our tests - first regular one and run tests on them and after that we do `build:offline` and run tests again.

Missing env vars in one of them will case our tests to fail

```suggestion
    "build:offline": "cross-env GATSBY_PREFIXED_FROM_COMMAND_LINE=YES FROM_COMMAND_LINE=YES TEST_PLUGIN_OFFLINE=y CYPRESS_SUPPORT=y gatsby build",
```

---

Ah! I see. Thanks for spotting that!